src/mono/mono/mini/mini-amd64.c

```diff
@@ -53,6 +53,9 @@
 MONO_DISABLE_WARNING(4127) /* conditional expression is constant */
 
 static GENERATE_TRY_GET_CLASS_WITH_CACHE (math, "System", "Math")
+static GENERATE_TRY_GET_CLASS_WITH_CACHE (swift_error, "System.Runtime.InteropServices.Swift", "SwiftError")
```

Comment:
Might be a good idea to move these to mini.c so its shared with mini-arm64.c.

Reply:
If moved to the `mini.c`, would it be still defined and usable in `mini-<arch>.c`? I assume it would require a declaration.

Reply:
Could we move it to `mini.h` instead of `mini.c`? Then it should get included for both mini and interpreter, right?

Reply:
Moved to `marshal.h` so that it can be accessed byt both `marshal.c` and `mini-<arch>.c`.